File: PWRSyllabus.Core/Entities/MinisterialEffect.cs
using System.Collections.Generic;
using PWRSyllabus.Core.Enums;

namespace PWRSyllabus.Core.Entities
{
    public class MinisterialEffect : BaseEntity
    {
        public string Description { get; set; }
        public string Code { get; set; }
        public Profile Profile { get; set; }
        public Level Level { get; set; }
        public FieldOfKnowledge FieldOfKnowledge { get; set; }
        public int FieldOfKnowledgeId { get; set; }
        public List<MinisterialEffectEducationalEffect> MinisterialEffectEducationalEffects { get; set; }

    }
}

File: PWRSyllabus.API/ClientApp/src/models/MinisterialEffect.ts
export default interface MinisterialEffect {
    id: number;
    code: string;
    discipline: string;
    category: string;
    descriptiveCategory: string;
    level: string;
    description: string;
}

const DefaultMinisterialEffect: MinisterialEffect = {
    id: 0,
    code : '',
    discipline : '',
    category: '',
    description: '',
    descriptiveCategory: '',
    level: '',
};

export {DefaultMinisterialEffect, MinisterialEffect};
